### PR TITLE
Nri story stuff ig

### PR DESCRIPTION
**Pull request by Stalkeros2**
## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
It's basically done but i gtg so I don't have time to write the description.

**Comment by Stalkeros2** [comment]
Too freaking busy these days so not like I can add a description; merg pls.

File: modular_skyrat/modules/stories_system/code/story_datums/very_impactful.dm [hunk 1]
```diff
@@ -10,92 +10,6 @@
 	impact = STORY_VERY_IMPACTFUL
 
 /*
-<<<<<<< HEAD
```
Comment (Zonespace27) [inline modular_skyrat/modules/stories_system/code/story_datums/very_impactful.dm:13]: merge skew

File: modular_skyrat/modules/stories_system/code/story_datums/unimpactful.dm [hunk 2]
```diff
@@ -101,70 +101,6 @@
 	)
 	maximum_execute_times = 1
 
-<<<<<<< HEAD
```
Comment (Zonespace27) [inline modular_skyrat/modules/stories_system/code/story_datums/unimpactful.dm:104]: merge skew

File: modular_skyrat/modules/stories_system/code/story_datums/somewhat_impactful.dm [hunk 3]
```diff
@@ -82,20 +102,6 @@
 	poor_sod = null
 	return ..()
 
-<<<<<<< HEAD
-/datum/story_type/somewhat_impactful/paradigm_shift
```
Comment (Zonespace27) [inline modular_skyrat/modules/stories_system/code/story_datums/somewhat_impactful.dm:86]: merge skew

File: modular_skyrat/modules/stories_system/code/story_datums/incredibly_impactful.dm [hunk 4]
```diff
@@ -7,25 +7,6 @@
 /datum/story_type/incredibly_impactful
 	impact = STORY_INCREDIBLY_IMPACTFUL
 
-<<<<<<< HEAD
```
Comment (Zonespace27) [inline modular_skyrat/modules/stories_system/code/story_datums/incredibly_impactful.dm:10]: merge skew

File: modular_skyrat/modules/stories_system/code/story_datums/incredibly_impactful.dm [hunk 5]
```diff
@@ -45,15 +26,9 @@
 	name = "Tourism Overload"
 	desc = "A swarm of up to 15 tourists come to the station."
 	actor_datums_to_make = list(
->>>>>>> e3bef5a336e7f5d7c1682f18c0878d79e8acbf4c
```
Comment (Zonespace27) [inline modular_skyrat/modules/stories_system/code/story_datums/incredibly_impactful.dm:48]: merge skew

File: modular_skyrat/modules/novaya_ert/code/head.dm [hunk 6]
```diff
@@ -19,6 +19,33 @@
 	desc = "Za rodinu!!"
 	armor = list(MELEE = 40, BULLET = 35, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, FIRE = 20, ACID = 50, WOUND = 20)
 
+/obj/item/clothing/head/hats/nri_revisor
+	name = "collegial servant hat"
+	desc = "The radio is as decorative as the hat is. Alt-click to adjust."
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/head.dm]: have the alt click part be on examine() and have "alt-click" be bold

File: modular_skyrat/modules/novaya_ert/code/head.dm [hunk 7]
```diff
@@ -19,6 +19,33 @@
...
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "nri_revisor"
+	inhand_icon_state = "that"
+	flags_inv = 0
+	strip_delay = 60
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/head.dm]:
```suggestion
	strip_delay = 6 SECONDS
```

File: modular_skyrat/modules/novaya_ert/code/head.dm [hunk 8]
```diff
@@ -19,6 +19,33 @@
...
+	/// What position the helmet is in, TRUE = DOWN, FALSE = UP
+	var/helmet_position = TRUE
+
+/obj/item/clothing/head/hats/nri_revisor/AltClick(mob/user)
+	. = ..()
+	helmet_position = !helmet_position
+	to_chat(user, span_notice("You flip the nasal breathing apparatus [src] [helmet_position ? "down" : "up"]."))
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/head.dm]: balloon alert

File: modular_skyrat/modules/novaya_ert/code/suit.dm [hunk 9]
```diff
@@ -167,3 +167,13 @@
...
+
+/obj/item/clothing/suit/toggle/nri_revisor
+	name = "collegial servant jacket"
+	desc = "An incredibly formal, incredibly thin jacket laced with gold trimming, used by the collegial servants of the NRI whenever yet another station needs to be inspected."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/suit_digi.dmi'
+	icon_state = "nri_revisor"
+	inhand_icon_state = "hostrench"
+	strip_delay = 50
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/suit.dm]:
```suggestion
	strip_delay = 5 SECONDS
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by Stalkeros2** [comment]
Gonna work on it somewhere around tomorrow as I'm still fairly busy.

File: modular_skyrat/modules/stories_system/code/story_datums/very_impactful.dm (hunk 1 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/stories_system/code/story_datums/very_impactful.dm:13]: Master branch moment; nothing I can do.

**Comment by Stalkeros2** [comment]
Wait a fucking second.

**Comment by Stalkeros2** [comment]
Oh my fucking god.

**Pull request closed without merging**